specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json

```diff
@@ -686,6 +686,10 @@
           "items": {
             "type": "string"
           }
+        },
+        "supportCredentials": {
```

Comment:
Why are these changes being made to an existing stable version?

Reply:
Changes to stable released versions must be approved by ARM

Reply:
These changes are additive. We do not want to release a new API-version for small non-breaking changes.

Reply:
Yes, missed that. Since these are additive changes I would guess these are not breaking changes.